drivers/wifi/esp_at/Kconfig.esp_at

```diff
@@ -154,6 +154,12 @@ config WIFI_ESP_AT_VERSION_2_1
 
 endchoice
 
+config WIFI_ESP_AT_CIPDINFO
+    bool "Enable CIPDINFO"
+    default n
```

Comment:
no need

Reply:
I have split up the pr. Will remove the esp_at.

Reply:
When CONFIG_WIFI_ESP_AT_CIPDINFO is enabled, the format of the network message will be changed to the following: +IPD,\<link_id\>,\<length\>,\<\"remote_ip\"\>,\<remote_port\>:\<data\>
The cmd_ipd_parse_hdr function can't parse the IP address, port, etc. In this case, it may cause packets to be dropped. So cmd_ipd_parse_hdr needs to be modified.

Reply:
@hongquan-prog I will take this in considerations when a make my PR for the esp_at part. Thank you.